.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - stable
+
```

Comment:
Could you add caching here? https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Caching-with-npm

Reply:
Ok